src/dataloader/dataloader.py

```diff
@@ -16,19 +16,19 @@
 
 
 class APTOSDataset(Dataset):
-    def __init__(self, data_path, total_image_num, train=True):
+    def __init__(self, data_path, total_image_num, type='train'):
```

Comment:
What values can this `type` parameter take? 
Is it included in the `params.json` file?

Reply:
We don't need to mention about `type` in `params.json`. However I will add about it in the README Dataloader section. `type` can take `train`, `val` or `test` values.